ansible/roles/basic_users/filter_plugins/remove.py

```diff
@@ -0,0 +1,19 @@
+""" Filter a dict to remove specified keys """
+
+import copy
+
+
+class FilterModule(object):
+
+    def filters(self):
+        return {
+            'remove_keys': self.remove_keys
+        }
+
+    def remove_keys(self, orig_dict, keys_to_remove):
```

Comment:
remove suggests that the original dict is modified. Would filter_keys be a better name?